src/controlnet_aux/lineart_anime/__init__.py

```diff
@@ -141,7 +142,11 @@ def to(self, device):
         self.model.to(device)
         return self
     
-    def __call__(self, input_image, detect_resolution=512, image_resolution=512, return_pil=True):
+    def __call__(self, input_image, detect_resolution=512, image_resolution=512, return_pil=None, output_type="pil"):
```

Comment:
Would do the same changes as in https://github.com/patrickvonplaten/controlnet_aux/pull/49/files#r1214565652

To deprecate a function argument we must remove it from the signature